framework/src/modules/random/types.ts

```diff
@@ -16,11 +16,13 @@ export interface ModuleConfig {
 	maxLengthRevealsMainchain?: number;
 }
 
-export interface UsedHashOnion {
+export interface UsedHashOnionInfo {
```

Comment:
Is the name for the interface acceptable, any suggestions for changes? Preferred to create a new interface for two fields as they are being referred enough times to merit for an interface definition.